conf/plugin-templates/common/dfs-storage-plugin.template

```diff
@@ -572,6 +572,9 @@
       ],
       "extractHeader": true,
       "delimiter": ","
+      },
+      "maprdb": {
```

Comment:
We use a different plugin for maprdb. I guess it's mfs or something. Can you use that instead?